src/palindrom.js

```diff
@@ -248,6 +248,52 @@ var Palindrom = (function() {
       }
     });
   }
+  /**
+   * Traverses/checks value looking for out-of-range numbers, throws a RangeError if it finds any
+   * @param {*} val value 
+   * @param {string} direction the inspected patch direction "outgoing"|"incoming"
+   */
+  PalindromNetworkChannel.prototype.findRangeErrors = function(val, direction) {
+    var type = typeof val;
+    if (type == 'object') {
+      for (var prop in val) {
+        if (val.hasOwnProperty(prop)) {
+          this.findRangeErrors(val[prop], direction);
+        }
+      }
```

Comment:
We are in ES6, right? Then I think something like `Object.values(val).forEach((val) => this.findRangeErrors(val[prop], direction));` could be a better option